terraform/worker.tf

```diff
@@ -0,0 +1,29 @@
+resource "cloudfoundry_app" "beis-roda-worker" {
+  name              = "beis-roda-${var.environment}-worker"
+  space             = cloudfoundry_space.space.id
+  instances         = 1
+  strategy          = "blue-green-v2"
+  command           = "bundle exec sidekiq"
+  health_check_type = "none"
+  memory            = 1024
+  disk_quota        = 3072
+  timeout           = 120
+  docker_image      = "thedxw/beis-report-official-development-assistance:${var.docker_image}"
+  service_binding { service_instance = cloudfoundry_service_instance.beis-roda-redis.id }
+  service_binding { service_instance = cloudfoundry_service_instance.beis-roda-postgres.id }
+  service_binding { service_instance = cloudfoundry_user_provided_service.papertrail.id }
+  environment = {
```

Comment:
These environment variables are the same when provided to both the web server and the worker. I presume there's no easy way to define them once and reuse them? We'd be less likely to forget to update the Sidekiq copy when adding new or changing old variables. Only if it's easy, not essential.

Reply:
when changing they are shared. since they use terraform variables which are set from the deploy process. Adding new ones is the problem. There is a way with locals I think.  I think if we add another env var we should look at the refactor then.